pyprocessing/renderer/tkinter/window.py

```diff
@@ -1,19 +1,22 @@
 from collections import deque
 import tkinter as tk
+from math import tau, degrees
```

Comment:
imports should be in alphabetical order, i.e. import math should be between import collections and import tkinter and degrees before tau